### DISCUSSION TITLE
Prettier documentation

### DISCUSSION THREAD

**Discussion by JonasPf** [zed-industries/zed#17297] [Feature Requests]
In the default settings I have the following section:

```
    "JavaScript": {
      "language_servers": ["!typescript-language-server", "vtsls", "..."],
      "prettier": {
        "allowed": true
      }
    },
```

I can't find this specific syntax for prettier in the docs. All I can find is how to configure it using `external`: https://zed.dev/docs/languages/javascript?highlight=prettier#code-formatting

Can you help me understand the differences or point me to the right documentation?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
